R/series-series.R

```diff
@@ -46,6 +46,20 @@ wrap.PlRSeries <- function(x, ...) {
   makeActiveBinding("dtype", function() self$`_s`$dtype() |> wrap(), self)
   makeActiveBinding("name", function() self$`_s`$name(), self)
   makeActiveBinding("shape", function() c(wrap(self$`_s`$len()), 1L), self)
+  makeActiveBinding(
+    "flags",
+    function() {
+      out <- list(
+        SORTED_ASC = self$`_s`$is_sorted_ascending_flag(),
+        SORTED_DESC = self$`_s`$is_sorted_descending_flag()
+      )
+      if (inherits(self$dtype, "polars_dtype_list")) {
+        out[["FAST_EXPLODE"]] <- self$`_s`$can_fast_explode_flag()
+      }
+      out
+    },
```

Comment:
Shouldn't this return a logical vector instead of a list?

Reply:
We could, yes. No strong preference on this

Reply:
Thanks, I understand. There is no need to use list here, so please use logical.

Reply:
Done